app/services/organizations/deselect_invoice_custom_section_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Organizations
+  class DeselectInvoiceCustomSectionService < BaseService
+    def initialize(section:)
+      @section = section
+      @organization = section.organization
+      super
+    end
+
+    def call
+      deselect_for_organization
+      result
+    end
+
+    private
+
+    attr_reader :section, :organization
+
+    def deselect_for_organization
+      return unless organization.selected_invoice_custom_sections.include?(section)
+
+      InvoiceCustomSectionSelection.where(
+        organization_id: organization.id, invoice_custom_section_id: section.id
```

Comment:
does the `organization_id` here represent this section is applied to the organization?

this is a little bit confusing as we use organization_id for multi tenancy purposes... I would go for a `applied_to_organization` flag instead...

Reply:
not exactly, the `organization_id` represents belonging to an organization, as organization can have multiple `invoice_custom_sections`, but only few of them will be selected on the organization level, and others (and the same) might be selected on the customer level

Reply:
and these invoice_custom_section_selections are exactly serving the purpose of choosing an invoice_custom_section for the organization or for the customer